doc/examples/applications/plot_solidification_tracking.py

```diff
@@ -0,0 +1,155 @@
+image_deltas = images_smoothed[1:, :, :] - images_smoothed[:-1, :, :]
+
+fig = px.imshow(
+    image_deltas,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Clip lowest and highest intensities
+# ===================================
+# We now calculate the 5th and 95th percentile intensities of ``image_deltas``:
+# We want to clip the intensities which lie below the 5th percentile
+# intensity and above the 95th percentile intensity, while also rescaling 
+# the intensity values to [0, 1]. 
+
+p_low, p_high = np.percentile(image_deltas, [5, 95])
+clipped = image_deltas - p_low
+clipped[clipped < 0.0] = 0.0
+clipped = clipped / p_high
+clipped[clipped > 1.0] = 1.0
+
+fig = px.imshow(
+    clipped,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Invert and denoise
+# ==================
+# Next, we invert the ``clipped`` images so the regions of highest intensity
+# will correspond to the region we are interested in tracking (i.e., the 
+# S-L interface). We now apply a total variation denoising filter to reduce 
+# noise beyond the interface.
+
+inverted = 1 - clipped
+denoised = restoration.denoise_tv_chambolle(inverted, weight=0.15)
+
+fig = px.imshow(
+    denoised,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig) 
+
+#####################################################################
+# Binarize
+# ========
+# Our next step is to create binary images, splitting each image 
+# into a foreground and a background: We want the S-L interface 
+# to be the most prominent feature in the foreground of each binary image,
+# so that it can eventually be separated from the rest of the image.
+# 
+# We need 
+# a threshold value ``thresh_val`` to create our binary images, ``binarized``. 
+# This can be set manually, but we'll use an automated minimum threshold 
+# method from the ``filters`` submodule of scikit-image (there are other 
+# methods that may work better for different applications).
+
+thresh_val = filters.threshold_minimum(denoised)
+binarized = denoised > thresh_val
+
+fig = px.imshow(
+    binarized,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Filter minimum size
+# ===================
+# In our binary iamges, the S-L interface appears as the largest region of
+# connected pixels. We can select this region by first labeling each separate
+# region in the binary images. This will create an image in which each pixel
+# coonected to other pixels of the region will be assigned a separate integer
+# value in ``labeled``. 
```

Comment:
I'm writing 'n-dimensional' loosely here, since the third dimension is not in space but in time.

Reply:
Okay I will make this change in my next commit! I realized we were both working at the same time a bit ago so I thought it would be best if I stepped away to let you finish reviewing 😅 I'll come back to this in a couple hours!